executor_manager/executors/docker/executor.py

```
@@ -307,6 +329,29 @@ def _create_new_container(
+
+                    tracker = get_running_task_tracker()
+                    tracker.add_running_task(
+                        task_id=task_id,
+                        subtask_id=task_info["subtask_id"],
+                        executor_name=executor_name,
+                        task_type=task.get("type", "online"),
+                    )
+                    logger.debug(
+                        f"Registered task {task_id} to RunningTaskTracker for heartbeat monitoring"
+                    )
+                except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)